### DISCUSSION TITLE
Catchment delineation

### DISCUSSION THREAD

**Discussion by pascalhorton** [mhm-ufz/mHM#96] [Q&A] [answered]
Hello,

We are setting up mHM for a catchment in Switzerland. As it's our first time, we have a few questions:

1. The lower part of the catchment is in a plain and therefore it is quite difficult for flow accumulation algorithms to delineate it properly. We do have a shapefile of the catchment delineated by someone else. However, if we use this one, we'll have some flow direction cells going out of the catchment or coming in. Should the catchment delineation (watershed mask) be fully consistent with the flow direction/accumulation or can some inconsistencies be tolerated (not on the main streams).

2. There is no discharge gauge station at the outlet of the catchment of interest, but we do have 3 gauge stations within the catchment. Can we use these nested catchments time series (covering most of the area) for the calibration while being interested in the overall catchment simulated discharge?

Thanks for your help!

Pascal

**Comment by pallavhydro** [comment, marked as answer]
Hi @pascalhorton 

Thanks for approaching and your questions. Following are my comments:

1. You can run even a rectangular block without issues with mHM. mHM has been used to run continents, where many large rivers have different outflow to the sea. Using the available catchment in your case should be fine. However, just an additional comment: you say the plain area is hard to delineate, which means preparing DEM derivatives such as flow accumulation (facc) might also be tricky (??). Or do you already have facc ready for you? As you might already know, facc is one of the required input for mHM. 

2. This is totally fine. mHM can have 1 or more gauges, situation at outlet or within. Since the inner gauges cover major portion of the catchment, the simulated discharge at the outlet, after calibration, can be considered to have sufficient reliability. A plus point in this situation is that the inner gauges also check local catchment behaviours resulting in better constraining of catchment properties -- a clear advantage of distributed models like mHM.

Hope this helps. Do let us know your progress! :)

Pallav

**Reply by pascalhorton** [reply]
Great, thanks!
Yes, achieving the correct flow accumulation was tricky, but we managed to force it to follow the actual river locations. Reconditioning using Arc Hydro Tools was not sufficient.
Thanks for helping!
